tutorials/tutorials.batch

```diff
@@ -123,117 +99,91 @@ rm -r /tmp/msui_tutorials/*
-# crop
-ffmpeg -y -i tmp1.mp4 -filter_complex "[0:v]crop=1920:1044:0:0[cropped]" -map "[cropped]" -crf 30 -acodec mp2 tmp2.mp4
-# scale
-# ffmpeg -y -i tmp2.mp4 -vf scale=1280:720 -c:v libx264 -crf 30 -acodec mp2 tmp3.mp4
-# set speed (very fast)
-# ffmpeg -y -i tmp3.mp4 -filter:v "setpts=0.1*PTS" tmp5.mp4
-# makes a gifcycle, based on tmp2.mp4, using max_colors=16[p] gives 3.1MByte gif file
-ffmpeg -y -i tmp2.mp4 -filter_complex "fps=5,scale=854:-1:flags=lanczos,split[s0][s1];[s0]palettegen=max_colors=32[p];[s1][p]paletteuse=dither=bayer" tutorial_waypoints.gif
+ffmpeg -y -i last_recording.mp4 -ss 00:00:24 -to 00:02:35 -c:v libx264 -crf 30 -acodec mp2 tmp1.mp4
+# crop action
+ffmpeg -y -i tmp1.mp4 -filter_complex "[0:v]crop=1100:900:1400:650[cropped]" -map "[cropped]" -crf 30 -acodec mp2 tutorial_waypoints.mp4
 
 # get back to tutorials dir
 cd ..
 
 ################################################################
-# views tutorial
+# views tutorial, 4K
 ~/bin/highlight-pointer -r 10 --key-quit q &
-$HOME/mambaforge/envs/mssdev/bin/python tutorial_views.py
+$HOME/Miniforge/envs/mssdev/bin/python tutorial_views.py
 
 # remove config files created for this tutorial
 rm -r /tmp/msui_tutorials/*
 
 # improve recording file using the copy last_recording.mp4
 cd recordings
 # slice (start and ending)
-ffmpeg -y -i last_recording.mp4 -ss 00:00:14 -to 00:09:14 -c:v libx264 -crf 30 -acodec mp2 tmp1.mp4
-# crop
-ffmpeg -y -i tmp1.mp4 -filter_complex "[0:v]crop=1920:1044:0:0[cropped]" -map "[cropped]" -crf 30 -acodec mp2 tmp2.mp4
-# scale
-# ffmpeg -y -i tmp2.mp4 -vf scale=1280:720 -c:v libx264 -crf 30 -acodec mp2 tmp3.mp4
-# set speed (very fast)
-# ffmpeg -y -i tmp3.mp4 -filter:v "setpts=0.1*PTS" tmp5.mp4
-# makes a gifcycle, based on tmp2.mp4, using max_colors=16[p] gives 3.1MByte gif file
-ffmpeg -y -i tmp2.mp4 -filter_complex "fps=5,scale=854:-1:flags=lanczos,split[s0][s1];[s0]palettegen=max_colors=32[p];[s1][p]paletteuse=dither=bayer" tutorial_views.gif
+ffmpeg -y -i last_recording.mp4 -ss 00:00:14 -to 00:08:22 -c:v libx264 -crf 30 -acodec mp2 tmp1.mp4
+# crop action result = 3K
+ffmpeg -y -i tmp1.mp4 -filter_complex "[0:v]crop=2560:1440:1200:650[cropped]" -map "[cropped]" -crf 30 -acodec mp2 tmp2.mp4
+# resize to HD
+ffmpeg -y -i tmp2.mp4 -vf scale=1920:1080 -c:v libx264 -crf 30 -acodec mp2 tutorial_views.mp4
 
 # get back to tutorials dir
 cd ..
 
 ####################################################
-# tutorial mscolab
+# tutorial mscolab, 3K, because we need the menu in full screen
 
 # start a mscolab server on standard port after you have it seeded
 # we should have a seed for tutorials
-$HOME/mambaforge/envs/mssdev/bin/python ../mslib/mscolab/mscolab.py db --seed
+$HOME/Miniforge/envs/mssdev/bin/python ../mslib/mscolab/mscolab.py db --seed
 
-$HOME/mambaforge/envs/mssdev/bin/python ../mslib/mscolab/mscolab.py start &
+$HOME/Miniforge/envs/mssdev/bin/python ../mslib/mscolab/mscolab.py start &
```

Comment:
My shell instincts tell me that this process will be left running after this script finished, but I would have to double check if that is true.

Reply:
yes,  it needs to be killed later. That can also become improved.